docs/@aptos-labs/ts-sdk-latest/index.md.bak

```diff
@@ -0,0 +1,5 @@
+---
+title: Latest Aptos TypeScript SDK Documentation
+permalink: /@aptos-labs/ts-sdk-latest
+redirect_to: /@aptos-labs/ts-sdk-1.7.0
+---
```

Comment:
Do you know why there's a bak file now?

I spent a bit of time trying to get `ts-sdk-latest`, and it should have updated via the script afaik